service/src/main/java/com/ge/predix/acs/monitoring/AcsMonitoringRepository.java

```diff
@@ -16,27 +16,33 @@
 
 package com.ge.predix.acs.monitoring;
 
-import javax.sql.DataSource;
-
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.jdbc.core.JdbcTemplate;
 import org.springframework.stereotype.Repository;
 
+import javax.sql.DataSource;
+
 /**
  *
  * @author 212360328
  */
 @Repository
 public class AcsMonitoringRepository {
 
+    private static final Logger LOGGER = LoggerFactory.getLogger(AcsMonitoringRepository.class);
+
     private JdbcTemplate jdbcTemplate;
 
     @Autowired
     public void setDataSource(final DataSource dataSource) {
         this.jdbcTemplate = new JdbcTemplate(dataSource);
     }
 
-    public void checkDBAccess() {
-        this.jdbcTemplate.queryForObject("select count(id) from policy_set", Integer.class);
+    void queryPolicySetTable() {
+        String query = "select count(id) from policy_set";
```

Comment:
This will be expensive , we can use telnet port or else select id from policy_set limit 1;